src/mpyl/cli/build.py

```diff
@@ -262,11 +272,34 @@ def by_choice(choice: Choice):
         return release.tag_name
 
 
+def get_test_releases():
+    url = "https://test.pypi.org/pypi/mpyl/json"
+    data = requests.get(url, timeout=30).json()
+    versions = list(data["releases"].keys())
+    versions.sort(key=LooseVersion, reverse=True)
+    versions = [version[:-4] for version in versions[:100]]
+    versions = list(dict.fromkeys(versions))
+    versions = [version + "*" for version in versions]
+    return versions
+
+
+def select_version() -> str:
+    console = Console()
+    console.log("Fetching MPyL releases..")
```

Comment:
I would put this log in the function above (since that actually fetches the releases)